client/sites-dashboard-v2/index.tsx

```diff
@@ -62,6 +70,9 @@ const SitesDashboardV2 = ( {
 	initialDataViewsState.search = search ?? '';
 	const [ dataViewsState, setDataViewsState ] = useState< DataViewsState >( initialDataViewsState );
 
+	// Filter sites list on search query
+	const filteredSites = useSitesListFiltering( allSites, { search: dataViewsState.search } );
```

Comment:
Im using our preexisting hook here contrary to @cleacos's initial recommendation to add this functionality in a `useEffect`. This hook already has the functionality we need and leverages other hooks for memoization and fuzzy searching. It doesn't yet seem necessary to reimplement this within a useEffect which doesn't support hooks, but if we find other blockers with this approach while implementing other controls (status filtering, pagination, sort, etc.) we may explore that option.

Reply:
That's great Addison!

Wiring the `dataViewsState.search` state to get the filtered sites using that hook is perfect 👌. I'm not familiar with that Dotcom stuff.